server/schedule/filter/filters.go

```diff
@@ -329,6 +330,12 @@ func (f *StoreStateFilter) isBusy(opt *config.PersistOptions, store *core.StoreI
 	return !f.AllowTemporaryStates && store.IsBusy()
 }
 
+func (f *StoreStateFilter) isOnStartApplyBusy(opt *config.PersistOptions, store *core.StoreInfo) bool {
+	f.Reason = "on-start-apply-busy"
+	return !f.AllowTemporaryStates && (store.GetUptime() < time.Second*5 ||
+		store.IsApplyBusy() && store.GetUptime() < time.Second*600)
```

Comment:
```suggestion
		store.IsApplyBusy() && store.GetUptime() < time.Minute*10)
```

Reply:
BTW, I suggest adding a comment for preventing transferring leaders in the first 5 seconds.

Reply:
@Connor1996 Could you please address this comment so we can merge this PR?

Reply:
Need to test the effectiveness first